### PR TITLE
Remove eps argument from fixed_batch_normalization method

### LINKED ISSUE #855: eps is a redundant argument of fixed_batch_normalization
`eps` is used only when `BatchNormalization.forward` is called with three arguments. So `fixed_batch_normalization` does not need `eps` because it calls forward propagation with five arguments.

### PR DESCRIPTION
**Pull request by unnonouno**
I removed eps argument of fixed_batch_normalization method.

fix #855

### CODE REVIEW

**Comment by beam2d** [comment]
I found that cudnnBatchNormalizationForwardInference (called at https://github.com/pfnet/chainer/blob/master/chainer/functions/normalization/batch_normalization.py#L152) does require the epsilon value. Is it ok to fix this value to the default one? I didn't assess how this argument is used by the cuDNN API.

**Comment by beam2d** [comment]
I found that the above issue is a bug of the current Chainer's implementation. It does not add eps in the test mode, but cuDNN does, and I think the latter is a better behavior w.r.t. the formulation given in the original paper. I close this PR and ask @unnonouno to fix it for v1 instead.

**Comment by delta2323** [comment]
Have we already open the issue for the problem mentioned above?

**Comment by beam2d** [comment]
Not yet.
